src/cms/forms/pages/page_translation_form.py

```diff
@@ -137,8 +138,24 @@ def clean_text(self):
 
         if "<h1>" in text:
             raise ValidationError(
-                _("Use of Heading 1 style not allowed."),
+                _("Use of Heading 1 style is not allowed."),
                 code="no-heading-1",
             )
 
+    def clean_text2(self):
```

Comment:
Quote from the [Django Documentation](https://docs.djangoproject.com/en/2.2/ref/forms/validation/):
"The clean_\<fieldname\>() method is called on a form subclass – where \<fieldname\> is replaced with the name of the form field attribute."

In other words, please do not create a second method `clean_text2()`, but put all validation code for the `text` field in the `clean_text()`-method. Django will call them magically when `form.is_valid()` is executed. It's no big problem that the code after the first `ValidationError` isn't run - it will only happen very rarely that the text field contains both `<h1>`-tags and external links, and in this case it's fine to fix one error at a time (e.g. after the error "Use of Heading 1 style is not allowed." the user removes the `<h1>`-tags, tries to submit the form again, gets the error "Use of external links is not allowed!" and removes the external links).